build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:8.2.1'
+        classpath 'com.android.tools.build:gradle:8.1.4'
```

Comment:
what's the need of this?

Reply:
Actually, due to some dependency issues, my Android Studio is giving an error when trying to build the project. The project is using an incompatible version (AGP 8.2.1) of the Android Gradle plugin. The latest supported version is AGP 8.1.4. That's why I changed it.

Reply:
Thanks for the clarification!